src/components/taskDetails/index.tsx

```diff
@@ -317,6 +331,14 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
                                     )}
                                 />
                             </TaskContainer>
+                            <TaskContainer
+                                hasImg={false}
+                                title="Request for task"
+                            >
+                                <button onClick={taskRequestHandle}>
+                                    Request for task
```

Comment:
Can we move this "Request for task" in the constants file?

Reply:
This label is currently not planned to be reusable, so I don't think there's a need to create a constant that might have a possibility of not getting used at all